kadasrouting/gui/navigationpanel.py

```diff
@@ -282,9 +335,13 @@ def updateNavigationInfo(self):
                 self.setWidgetsVisibility(True)
                 self.setWarnings(instructions["raw_distleft"])
             else:
-                self.setMessage(self.tr("Select a route or waypoint layer for navigation"))
+                self.setMessage(self.tr("The 'Routes' layer has no waypoints. Hint: save your project."))
```

Comment:
If the project is not saved, instead of warning the user, why not saving a copy of it to a temp folder and reading it from there? It would be an improvement in terms of user experience, I think. Or that could be done in any case, so if there are no waypoints found, that really means that the project does not contain any.

Reply:
good idea.

Reply:
done